auction-house/program/tests/execute_sale.rs

```diff
@@ -17,7 +17,7 @@ use mpl_testing_utils::{
     solana::{airdrop, create_associated_token_account, transfer},
     utils::Metadata,
 };
-use solana_sdk::signer::Signer;
+use solana_sdk::{commitment_config::CommitmentLevel, signer::Signer};
```

Comment:
These changes here and a lot more below seem totally unrelated to _removing Rust workspaces_ as are a lot in the files below.
So I'm a bit confused about those.

Reply:
I thought the Banks Client defaulting to using *Finalized* might be the issue behind some of the test time out issues we've seen in CI. Changing the commitment to *Confirmed* should resolve txs faster.